homework_29_280323/src/resources/init.sql

```diff
@@ -0,0 +1,45 @@
+CREATE database homework_29_280323;
+
+DROP table persons;
```

Comment:
read about if exists statement

Reply:
I've read and fixed (removed DROP statement).